packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -463,6 +464,12 @@ def check_statsd_metrics():
                         assert sample[2] == val
                         if len(expected_copy) == 0:
                             return
+            sys.stderr.write(
```

Comment:
Could we combine the stderr with the raise statement to a line like this
`assert expected_metrics == expected_copy, "Expected statsd metrics not found"`

Reply:
It's not quite that easy, as we want `expected_copy` to be empty by this stage.

The easiest is just to leave this out. This is just debugging code, to show the problem - the real fix is elsewhere.